### ISSUE TITLE
Rendering special characters with a combining character

### ISSUE THREAD

**Issue by Intelligent2013** [metanorma/metanorma-iso#636]
ISO-9 contains Cyrillic characters. Some of them are encoded as a special character + combining character.

Example:
`&#228;&#772;` (letter a with umlaut combined with a macron)
which is rendered properly in HTML as:
![image](https://user-images.githubusercontent.com/55683568/164995293-b7637011-f91d-4595-8e99-a80608299e56.png)

However, umlaut and macron overlap in PDF: 
![image](https://user-images.githubusercontent.com/55683568/164995384-375f22d3-963c-4f6d-93ed-171928c428d6.png)

**Comment by Intelligent2013** [comment]
I've opened ticket on Apache FOP tracker: https://issues.apache.org/jira/browse/FOP-3063

**Comment by Intelligent2013** [comment]
Another ticket opened for non-centered combining chars: https://issues.apache.org/jira/browse/FOP-3065

**Comment by Intelligent2013** [comment]
The fonts Cambria and 'Noto Sans/Serif' contains 'letter a with diaeresis and macron':
![image](https://user-images.githubusercontent.com/5559926/172372913-820abb18-b126-4380-b6ff-7ed696a07ae2.png)

So the sequence `&#228;&#772;` can be replaced to `&#479;`.

I'll try to find another characters replacements for ISO-9.

**Comment by Intelligent2013** [comment]
- #### Latin small letter a with diaeresis and dot below
```
&#228;&#803;
```
![image](https://user-images.githubusercontent.com/5559926/172404396-cf8add74-366b-4886-8adf-1599002d7ed1.png)

there isn't replacement in the fonts.

- #### Latin small letter z with caron and comma below
```
&#382;&#806;
```
![image](https://user-images.githubusercontent.com/5559926/172407899-5951bcd9-ae1f-4be8-ad0f-84801e13354b.png)

renders ok, no need replacement.

- #### Cyrillic small letter i with acute

```
&#x438;&#769;
```
![image](https://user-images.githubusercontent.com/5559926/172408188-b0294340-29f3-433e-9186-79f493a978a4.png)

there isn't replacement in the fonts.

- #### letter i with macron and dot above
```
&#299;&#775;
```
![image](https://user-images.githubusercontent.com/5559926/172409858-fe354923-f96b-4a82-9731-fb0fda8c3102.png)

there isn't replacement in the fonts.
- #### Cyrillic small letter letter ka with hook
```
&#x43A;&#807;
```
![image](https://user-images.githubusercontent.com/5559926/172412009-3023aa36-3ae8-4b9f-a439-bea2bd5222d0.png)

In Cambria  and Noto fonts there are glyph `&#1220`:
![image](https://user-images.githubusercontent.com/5559926/172412214-62e4b22a-7ef8-41a3-8b82-585d352ee493.png)

- #### character
```
&#x43A;&#808;
```
![image](https://user-images.githubusercontent.com/5559926/172412810-e29f7ca2-27bb-4e96-b9be-7498702810cf.png)

renders ok. No need replacement.

- #### the character
```
&#x43B;&#x315;
```
![image](https://user-images.githubusercontent.com/5559926/172413140-646b417d-0854-4b23-b642-5b13f8969ed0.png)

renders ok. No need replacement.

- #### character
```
| 52 | &#x43D;&#807;
```
![image](https://user-images.githubusercontent.com/5559926/172413540-df0134d1-3575-48b7-bc93-8d0ed271d277.png)

renders ok. No need replacement.

- #### character
```
| 54 | &#x43D;&#808;
```
![image](https://user-images.githubusercontent.com/5559926/172413691-7e712279-31e1-408c-b235-dfe846233526.png)

renders ok. No need replacement.

- #### Cyrillic small letter n with macron
```
| 57 | &#x43D;&#772;
```
![image](https://user-images.githubusercontent.com/5559926/172413897-e29bbd51-59c0-4fce-b561-ebf7ae64ba5f.png)

There isn't replacement in the fonts.

- #### Latin small letter o with diaeresis and macron
```
| 62 | &#333;&#776;
```
![image](https://user-images.githubusercontent.com/5559926/172414825-35161c6f-4aa3-4ffd-b414-cc23e1676369.png)

There is glyph '&#555':
![image](https://user-images.githubusercontent.com/5559926/172415042-f0a1d688-3a15-4f26-8b6e-448ee08585dd.png)

- #### Latin small letter o with diaeresis and dot below:
```
&#246;&#x323;
```

![image](https://user-images.githubusercontent.com/5559926/172415638-d41b5c68-0111-4334-87cd-0e43224fb49e.png)

There isn't replacement in the fonts.

- #### the character
```
| 75 | &#x442;&#807;
```

![image](https://user-images.githubusercontent.com/5559926/172416153-962130aa-4688-431f-8acd-d785a6a4a146.png)

renders ok.

- #### Latin small letter T with caron
```
| 76 | &#x442;&#780;
```
![image](https://user-images.githubusercontent.com/5559926/172416595-bf1483fc-4d7b-4341-a1f6-d45ecc36c026.png)

There isn't replacement.

- #### the character
```
| 77| ... &#x163;&#806;
```
![image](https://user-images.githubusercontent.com/5559926/172417528-42ed9041-caa8-4bfa-9932-251a7fd1fb71.png)

renders ok.
- #### the character
```
| 84 | &#563;&#776; | &#252;&#803;
```
![image](https://user-images.githubusercontent.com/5559926/172418356-35767fd7-73c3-4c64-8b11-d9dac74bcb3e.png)

There isn't replacement.

- #### the character
```
| 84 | ... | &#252;&#803;
```
![image](https://user-images.githubusercontent.com/5559926/172418516-85938d22-a041-4c26-a996-5d2b2ed1a8bf.png)

There isn't replacement.


- #### the character
```
| 87 | &#255;&#772; 
```
![image](https://user-images.githubusercontent.com/5559926/172419686-48d176d5-0328-455e-be6a-733791e52f62.png)

There isn't replacement.

- #### the character
```
| 87 | ... | &#363;&#803;
```
![image](https://user-images.githubusercontent.com/5559926/172419981-b33423fb-1dd7-4065-89df-2400a6517f60.png)

There isn't replacement.

- #### the character
```
| 97 | &#x447;&#791; 
```
![image](https://user-images.githubusercontent.com/5559926/172420146-367a967e-514a-4fe6-8f3e-c0889973e3a2.png)

should be replaced by `&#1228;`
![image](https://user-images.githubusercontent.com/5559926/172420366-b84f917b-6fd5-48d6-96cf-3eb965790e0d.png)

- #### the character
```
| 100 | &#x447;&#832;
```
![image](https://user-images.githubusercontent.com/5559926/172420587-5c3dbb64-9a27-4bbe-aa35-da95416fd3d6.png)

There isn't replacement.

- #### the character
```
| 102 | ... | c&#774;&#808;
```
![image](https://user-images.githubusercontent.com/5559926/172421058-aee622b5-72d2-466a-9e8a-45f96aaaa441.png)

renders ok.


- #### the character 
```
| 108 | &#x44B;&#773;
```

![image](https://user-images.githubusercontent.com/5559926/172421168-29d2dc4f-7f3a-4e50-8b1b-c3cf2ca5d4e0.png)

There isn't replacement.


```
| 114 | &#x44E;&#773;
```

**Comment by ronaldtse** [comment]
@Intelligent2013 for the characters that have better replacements, can you apply them to the ISO 9 document as well using a new issue? Thanks!

**Comment by Intelligent2013** [comment]
> @Intelligent2013 for the characters that have better replacements, can you apply them to the ISO 9 document as well using a new issue? Thanks!

@ronaldtse done in https://github.com/metanorma/iso-9/issues/11

**Comment by ronaldtse** [comment]
Thanks @Intelligent2013 for the update.

**Comment by Intelligent2013** [comment]
Issue fixed in mn2pdf v1.47 and latest xslt in metanorma-iso.

**Comment by Intelligent2013** [comment]
Closing as completed.
